### PR TITLE
Remove BO from release.ci

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed removes BO from release.ci. Least year, I've removed BO references from our release docs already.

### CODE REVIEW

File: plugins.txt [hunk 1]
```diff
@@ -123,7 +94,6 @@ pipeline-model-extensions:2.2151.ve32c9d209a_3f
 pipeline-rest-api:2.34
 pipeline-stage-step:305.ve96d0205c1c6
 pipeline-stage-tags-metadata:2.2151.ve32c9d209a_3f
-pipeline-stage-view:2.34
```
Comment (dduportal) [inline plugins.txt:126]: Can we keep this plugin, it provides useful metrics which are easy to get otherwise

Comment (NotMyFault) [reply plugins.txt:126]: Surely, but I thought it was superseded by the pipeline-graph-view plugin?

Comment (dduportal) [reply plugins.txt:126]: Yes, in theory. But this old plugin provides, at a glance, a view of the timing with build history. I'm not aware how to do this with alternatives without  changing pages multiple times.

Comment (NotMyFault) [reply plugins.txt:126]: Absolutely, there we go :)

**Review by dduportal — APPROVED** [review]

**Pull request merged**